src/Lifecycle.js

```diff
@@ -272,14 +275,19 @@ export function getLocalStorageSessionVars() {
 //      The plan is to gradually move the localStorage access done here into
 //      SessionStore to avoid bugs where the view becomes out-of-sync with
 //      localStorage (e.g. isGuest etc.)
-async function _restoreFromLocalStorage() {
+async function _restoreFromLocalStorage(ignoreGuest) {
```

Comment:
Could we also use an options object here as well, so that all callers are forced to specify it by name? Could do something like `{ ignoreGuest } = {}` so that it can still be called with no args.

Reply:
done!